app/routes/book_routes.py

```diff
@@ -60,6 +60,14 @@ def update_book(book_id):
 
     return Response(status=204)
 
+@books_bp.delete("/<book_id>")
+def delete_book(book_id):
+    book = validate_book(book_id)
+    db.session.delete(book)
+    db.session.commit()
+
+    return Response(status=204)
```

Comment:
Similar to the PUT case, I'd suggest code similar to the following to make sure the content-type is marked JSON, even though the body is empty. Otherwise, the content type is marked text/html.

In conjunction with the PUT case, this would allow not importing Response.

```suggestion
    return make_response({}, 204)  # 204 No Content
```